CMakeLists.txt

```diff
@@ -234,7 +234,7 @@ endif()
 
 set(BINARY_INST_DESTINATION "bin")
 set(RESOURCE_INST_DESTINATION "share/shadered")
-install(PROGRAMS bin/SHADERed DESTINATION "${BINARY_INST_DESTINATION}" RENAME shadered)
+install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/bin/SHADERed DESTINATION "${BINARY_INST_DESTINATION}" RENAME shadered)
```

Comment:
This should be done to all the hard-coded paths, e.g:
```cmake
install(DIRECTORY bin/data bin/templates bin/themes bin/plugins DESTINATION "${RESOURCE_INST_DESTINATION}")

if (UNIX AND NOT APPLE)
	# install XDG desktop file and icon on Linux/BSD
	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/Misc/Linux/org.shadered.SHADERed.desktop DESTINATION "share/applications" RENAME shadered.desktop)
	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/Misc/Icon/icon.svg DESTINATION "share/pixmaps" RENAME shadered.svg)
	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/bin/icon_32x32.png DESTINATION "share/icons/hicolor/32x32/apps" RENAME shadered.png)
	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/bin/icon_64x64.png DESTINATION "share/icons/hicolor/64x64/apps" RENAME shadered.png)
	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/bin/icon_128x128.png DESTINATION "share/icons/hicolor/128x128/apps" RENAME shadered.png)
	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/bin/icon_256x256.png DESTINATION "share/icons/hicolor/256x256/apps" RENAME shadered.png)
endif()
```

Reply:
very nice bro